core/src/unprocessed_packet_batches.rs

```diff
@@ -158,7 +176,7 @@ impl Ord for ImmutableDeserializedPacket {
 /// Currently each banking_stage thread has a `UnprocessedPacketBatches` buffer to store
 /// PacketBatch's received from sigverify. Banking thread continuously scans the buffer
 /// to pick proper packets to add to the block.
-#[derive(Default)]
+#[derive(Default, Debug)]
```

Comment:
nit: not sure if `Debug` is absolutely necessary here

Reply:
need it for the Debug on the encapsulating enum